docs/current_docs/features/visualization.mdx

```diff
@@ -7,34 +7,46 @@ description: "Visualize pipelines in real-time"
 
 Dagger works by building up a DAG of operations and evaluating them, often in parallel. By nature, this is a difficult thing to display. It's easy to show the output of one operation at a time, but as soon as you have parallelism, you'll get jumbled output that's hard to understand.
 
-This quickly creates the need to visualize the pipeline, to understand what's going on. Dagger includes two features for this purpose.
+This quickly creates the need to visualize your pipelines to understand and measure what's going on. Dagger includes two features for this purpose - a [Terminal UI](#terminal-ui) and [Dagger Cloud's Traces](#traces).
```

Comment:
Would it make sense to start transitioning from `pipeline` to `workflow` everywhere? Note: doesn't need to be part of this PR.

Reply:
Yeah, might be worth bringing up in our next call?